lte/cloud/go/services/nprobe/obsidian/models/swagger.v1.yml

```diff
@@ -0,0 +1,294 @@
+---
+swagger: '2.0'
+
+magma-gen-meta:
+  go-package: magma/lte/cloud/go/services/nprobe/obsidian/models
+  dependencies:
+    - 'orc8r/cloud/go/models/swagger-common.yml'
+    - 'orc8r/cloud/go/services/orchestrator/obsidian/models/swagger.v1.yml'
+  temp-gen-filename: lte-nprobe-swagger.yml
+  output-dir: lte/cloud/go/services/nprobe/obsidian
+  types:
+    - go-struct-name: NetworkProbeTaskID
+      filename: network_probe_task_id_swaggergen.go
+    - go-struct-name: NetworkProbeTaskDetails
+      filename: network_probe_task_details_swaggergen.go
+    - go-struct-name: NetworkProbeTask
+      filename: network_probe_task_swaggergen.go
+    - go-struct-name: NetworkProbeDestinationID
+      filename: network_probe_destination_id_swaggergen.go
+    - go-struct-name: NetworkProbeDestinationDetails
+      filename: network_probe_destination_details_swaggergen.go
+    - go-struct-name: NetworkProbeDestination
+      filename: network_probe_destination_swaggergen.go
+
+info:
+  title: LTE Network Probes Management
+  description: LTE REST APIs
+  version: 1.0.0
+
+basePath: /magma/v1
+
+paths:
+  /lte/{network_id}/network_probe_tasks:
```

Comment:
Rather than `network_probe_tasks`, I'd prefer making network_probe a subroute such that you have:

`/lte/{network_id}/network_probe/tasks`
and 
`/lte/{network_id}/network_probe/definitions`

Reply:
Sure, this is definitely better.